docker-compose.yml

```diff
@@ -11,7 +11,7 @@ services:
         - DB_PASSWORD=Ab132xw
         - DB_NAME=psql_db
         - DB_HOST=postgresql
-        - DB_PORT=5432
+        - DB_PORT=5032
```

Comment:
Why we changed port ?

Reply:
There were some conflicts between travis and DB. @mariawedler has more information about that.